go.mod

```diff
@@ -17,7 +18,7 @@ require (
 	google.golang.org/grpc/cmd/protoc-gen-go-grpc v1.3.0
 	google.golang.org/protobuf v1.30.0
 	k8s.io/api v0.26.3
-	k8s.io/apimachinery v0.26.3
+	k8s.io/apimachinery v0.27.1
```

Comment:
operator-sdk does not like this update. Can you prevent this change?

Reply:
yes I saw, we need to downgrade it first in vg repo, I will do it and will update here

Reply:
done